dio/lib/src/dio_event_processor.dart

```diff
@@ -13,10 +13,15 @@ class DioEventProcessor implements EventProcessor {
   static final _dioErrorType = (DioError).toString();
 
   /// This is an [EventProcessor], which improves crash reports of [DioError]s.
-  DioEventProcessor(this._options, this._maxRequestBodySize);
+  DioEventProcessor(
```

Comment:
Can we also add to the HttpClient integration?

Reply:
Yeah, but I would do that in another PR.

Reply:
https://github.com/getsentry/sentry-dart/issues/969